sentences/de/_common.yaml

```diff
@@ -0,0 +1,15 @@
+language: de
+responses:
+  errors:
+    no_intent: "Entschuldigung, das habe ich nicht verstanden"
+    no_area: "Den Bereich {{ area }} gibt es nicht"
+    no_domain: "{{ domain }} gibt es nicht in dem Bereich {{ area }}"
```

Comment:
```suggestion
    no_domain: "{{ domain }} gibt es nicht im Bereich {{ area }}"
```

Reply:
this merged version should only be used, when you talk about an not specific defined noun (_mostly a generic place description_), but when the noun is well known or more specific defined, then you should not merge it.

examples:
### merged version, because generic noun/place
- Ich war im Kino 
- Wir haben uns im Stadtzentrum getroffen

### not merged, because well known or more specific defined noun/place
- In dem Supermarkt würde ich nicht einkaufen
- In diesem Supermarkt würde ich nicht einkaufen
- In dem Winter 2010 war es sehrt kalt

Reply:
I understand your explanation. My suggestion is just a gut feeling because the sentence "Ventilator gibt es nicht in dem Bereich Küche" sounds strange compared to "Ventilator gibt es nicht im Bereich Küche".

Feel free to close the conversation without touching the PR.

Reply:
@spacegaier any toughs on this? 🙂

Reply:
I am leaning towards the proposal from syssi. I get your point from a purely grammatical point of view, but from a gut feeling perspective "in dem Bereich Küche" sounds actually a bit weird. And since we are talking about conversational language "im" feels more suitable to me.

Reply:
Ok, you both have convinced me